### PR TITLE
[201811] enable DB migrator code

### PR DESCRIPTION
**Pull request by yxieca**
**- What I did**
Enable the existing DB migration code

**- How to verify it**
warm boot from before PFC_WD table rename to after.

### CODE REVIEW

File: scripts/db_migrator.py [hunk 1]
```diff
@@ -68,9 +68,9 @@ def version_unknown(self):
         #       here we only intended to migrade to DB version 1.0.1.
         #       If new DB version is added in the future, the incremental
         #       upgrade will take care of the subsequent migrations.
-        # self.migrate_pfc_wd_table()
-        # self.set_version('version_1_0_1')
-        # return 'version_1_0_1'
+        self.migrate_pfc_wd_table()
+        self.set_version('version_1_0_1')
```
Comment (prsunny) [inline scripts/db_migrator.py]: is the version `1.0.1` or `1_0_1`? As per comments its `1.0.1`

Comment (yxieca) [reply scripts/db_migrator.py]: 1_0_1 because it is a string used for function name. '.' doesn't work too well there.

Comment (yxieca) [reply scripts/db_migrator.py]: 1_0_1 is reformatted version number for 1.0.1. Do you want me to change the comment?

**Comment by yxieca** [comment]
retest this please

File: scripts/db_migrator.py (hunk 1 above)
Comment (prsunny) [reply scripts/db_migrator.py]: That's fine.. just want to confirm

**Pull request merged**